Necrosis/Necrosis.lua

```diff
@@ -2126,31 +2126,31 @@ function Necrosis:StoneToRank(spellName)
 		return self.Translation.Misc.Rank .. " 1"
 	end
 	if (spellName:find(self.Translation.StoneRank.Major)) then
-		return self.Translation.Misc.Rank .. " 2"
+		return self.Translation.Misc.Rank .. " 5"
 	end
 	if (spellName:find(self.Translation.StoneRank.Lesser)) then
-		return self.Translation.Misc.Rank .. " 4"
+		return self.Translation.Misc.Rank .. " 2"
```

Comment:
Thanks for the fix. This fixes the issue where a lesser healthstone was created when a higher rank is available.

Reply:
That alone doesn't actually fix the healthstone problem. I fixed this first but it didn't change anything.

It's also the change of spellid in spells.lua from Healthstone (Lesser) to just Healthstone. Without changing that the ranking code looks for a higher rank of "Healthstone (Lesser)" which doesn't work.

Reply:
Seems to work thanks!